flow/layers/child_scene_layer.cc

```diff
@@ -4,15 +4,42 @@
 
 #include "flow/layers/child_scene_layer.h"
 
+#include "mojo/skia/type_converters.h"
+
 namespace flow {
 
+// TODO(abarth): We need to figure out how to allocate these ids sensibly.
+static uint32_t next_id = 10;
+
 ChildSceneLayer::ChildSceneLayer() {
 }
 
 ChildSceneLayer::~ChildSceneLayer() {
 }
 
+void ChildSceneLayer::Preroll(PrerollContext* context, const SkMatrix& matrix) {
+  transform_ = matrix;
+}
+
 void ChildSceneLayer::Paint(PaintContext::ScopedFrame& frame) {
 }
 
+void ChildSceneLayer::UpdateScene(mojo::gfx::composition::SceneUpdate* update,
+                                  mojo::gfx::composition::Node* container) {
+  uint32_t id = next_id++;
+
+  auto child_resource = mojo::gfx::composition::Resource::New();
```

Comment:
Yeah, I'm not entirely sure how to handle the bifurcation of what works in standalone Flutter versus what works in Mojo-land.  I'm happy to add a comment.